examples/README.md

```diff
@@ -136,3 +136,18 @@ to point to the correct `.js` file.
 Then serve `examples/wasm` dir to browser. i.e.
 
     $ basic-http-server examples/wasm
+
+## ios
+
+#### pre-req
+
+    $ rustup target add aarch64-apple-ios x86_64-apple-ios
+    $ brew install xcodegen
```

Comment:
I think id prefer it if we just check in the full project. I want to keep the "setup dependencies" as small as possible. We will likely eventually build a "bevy command line tool" for initializing the "glue" for the platforms that need it, and having the full project will help inform the construction of that tool.

How "big" is the xcode project output?

Reply:
Yeah, it's a fair. In a lot of other projects ([glutin](https://github.com/rust-windowing/glutin/tree/master/glutin_examples/ios-example), etc), I've just added the whole Xcode project and it's not too big.